test/data.jl

```diff
@@ -186,6 +192,16 @@ end
     @test eltype(df2.a) == Union{Int, Missing}
     @test df.a == df2.a == [1, 3]
 
+    # view=true
+    df = DataFrame(a=[1, missing, 3])
+    @test dropmissing(df, view=false) == DataFrame(a=[1, 3])
+    @test dropmissing(df, view=true) == view(df, [1, 3], :)
+    @test typeof(dropmissing(df, view=true)) <: SubDataFrame
+    @test eltype(dropmissing(df, view=true, disallowmissing=false).a) == Union{Int, Missing}
+    # @test_throws ArgumentError dropmissing(df, view=true, disallowmissing=true) # tested separately
```

Comment:
Why keep this commented out?

Reply:
This exact path is tested slightly lower in the kwarg test shared by dropmissing and unique, so I didn’t want to duplicate it, but equally I didn’t want to delete it to show that it was considered. It takes milliseconds, so I should probably just keep it.